libs/core/device-manager/services/usb-devices/usb-devices.helper.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { getUsbDevicesLinux } from "./usb-devices-linux.helper"
+import { getUsbDevicesMacOS } from "./usb-devices-mac.helper"
+import { getUsbDevicesWindows } from "./usb-devices-windows.helper"
+
+export async function getUsbDevices() {
```

Comment:
⚪ 
What do you think about handling exceptions from the helper functions here to avoid difficult-to-debug issues if they propagate?

Reply:
Do you mean normal try...catch approach? Yea, thats good idea.